packages/buildbucket-dart/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 1.0.9
+
+- Pin versions due to https://github.com/flutter/flutter/issues/110272
+
+# 1.0.8
+
+- Update dependencies.
```

Comment:
Shall we release one version each time by combining these two changes?

Reply:
So I had already deployed 1.0.8 by mistake. Which is why I updated to 1.0.9. I am not really sure what the process is concerning releasing packages either. In this case it was my mistake and I will wait until after the merge to release the new package.

Reply:
Is there a doc on the release process we have? Also I am wondering if there is some way to test the package locally before deploying. In Java you have the ability to do this but I am not sure about dart.